pkg/sqlite/conversion.go

```diff
@@ -0,0 +1,131 @@
+package sqlite
```

Comment:
Are we converting the sqllite db to model before serving the db anymore now that we've decided serving with configs is a separate command altogether? 

I couldn't find a place where we're doing that in this PR, maybe I'm missing something?

Reply:
You're correct. The `opm registry serve` command is completely unchanged, so sqlite DBs will be served directly (without converting to the model).

The reason this file is present in this PR is to enable the test in `./pkg/server/server_test.go`. In that test, we serve the DB directly, and we convert the DB to the model and serve that via the model-based GRPC server. This helps us guarantee that the sqlite GRPC server and the model-based GRPC server function the same against the same underlying data.